vertical-pod-autoscaler/pkg/utils/annotations/vpa_inplace_update.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package annotations
+
+// TODO(maxcao13): This annotation currently doesn't do anything. Do we want an annotation to show vpa inplace resized only for cosmetic reasons?
+
+const (
+	// VpaInPlaceUpdatedLabel is a label used by the vpa inplace updated annotation.
+	VpaInPlaceUpdatedLabel = "vpaInPlaceUpdated"
+)
```

Comment:
Is this a label or an annotation?

Reply:
I copied the style from here: https://github.com/kubernetes/autoscaler/blob/54fe60ed4d4bb4cb89fe4abe11284d1bd6b06390/vertical-pod-autoscaler/pkg/utils/annotations/vpa_observed_containers.go#L29. I'm not really sure of the intention of using the word `label` since it was written 6 years ago, but it probably makes more sense to be `key`.